.prettierrc.toml

```diff
@@ -0,0 +1,7 @@
+bracketSpacing = false
+plugins = ['prettier-plugin-toml']
+printWidth = 100
+semi = false
+singleQuote = true
+tabWidth = 4
```

Comment:
Not sure if this aleady covers yaml linting, but I recently discovered you can ask prettier to fix whitespace in yaml. I am not sure what the toml equivalent is, but the .prettierrc file looks like this 
```
{
    "tabWidth": 4,
    "useTabs": false
}
```

Reply:
Duncan bro, you gotta catch up

Reply:
Duncan bro, you gotta catch up

Reply:
GitHub ruined my joke